### ISSUE TITLE
decide which terminology to adopt (emacs or vscode)

### ISSUE THREAD

**Issue by MarcoIeni** [VSpaceCode/VSpaceCode#82]
For example if we decide to adopt emacs temiinology "Move editor group right" becomes "Move window right".

**Comment by MarcoIeni** [comment]
I mean, we should decide if we should follow emacs or vscode terminology and adopt it. For what I have understood:

```
| emacs  | vscode       |
|--------+--------------|
| buffer | editor       |
| window | editor group |
| frame  | window       |
```

**Comment by stevenguh** [comment]
I am not sure there's clear cut line here. I tend to use the vscode terminology and I have been using `/` to indicate emacs terminology when the key associate doesn't make sense with vscode terminology. Maybe having a unified naming scheme like the use of `/` or parenthesis to accompany both?

**Comment by stevenguh** [comment]
Just link a similar issue about the consolidating the keybinding for window/frame/proejct I commented on a commit here. https://github.com/VSpaceCode/VSpaceCode/commit/a845b06fa0286188348fbc067a62a44181a85543#commitcomment-41321674

**Comment by MarcoIeni** [comment]
> I am not sure there's clear cut line here. I tend to use the vscode terminology and I have been using `/` to indicate emacs terminology when the key associate doesn't make sense with vscode terminology. Maybe having a unified naming scheme like the use of `/` or parenthesis to accompany both?

So for example `SPC b u` from "Reopen closed editor" becomes "Reopen closed buffer/editor" and `SPC b d` from "Close active editor" becomes "Close active buffer/editor"? The downside is that it's a little bit too verbose :sweat_smile:

**Comment by stevenguh** [comment]
Yeah, now that I think more about it. I am thinking we should adopt more the emacs naming otherwise the binding doesn't make much sense and put the note on the readme to explain to the differences in terminology.

**Comment by MarcoIeni** [comment]
I would really like to hear user opinions about this!
Let's leave it open for one or two weeks and see if someone would like to share their thoughts on this :)

**Comment by stevenguh** [comment]
Makes sense 👍

**Comment by stevenguh** [comment]
Another point worth considering is that when we use/bundle other extensions, they are more likely to use vscode-like terminology.

**Comment by MarcoIeni** [comment]
Two weeks are gone and nobody answered..Steven nobody cares about us :sob: 

Jokes aside, I propose to:

- use emacs terminology on key binding name (i.e. "Move window right" instead of "Move editor group right")
- explain this choice in the readme by copy pasting [this](https://github.com/VSpaceCode/VSpaceCode/issues/82#issuecomment-671062974) table

**Comment by stevenguh** [comment]
hahahah....for real.

That sounds fine to me. I think there's always going to have something that's inconsistent, but I think the current table is small enough set that it make sense. Especially, if people are coming from spacemacs, they probably use that terminology more often.

**Comment by MarcoIeni** [comment]
solved in 90e93b8a356f01156bb60f5dd7b41dcfb60277ed
